.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,8 +1,14 @@
 blank_issues_enabled: true
 contact_links:
-  - name: ⁉️ Need help with Node.js?
+  - name: ⁉️ Need general help with Node.js?
     url: https://github.com/nodejs/help
     about: Please file an issue in our help repo.
+  - name: 📦 Have an issue with npm?
+    url: https://github.com/npm/cli/issues
+    about: npm has a seperate issue tracker.
+  - name: 💡 Have an issue with the JavaScript specification?
+    url: https://github.com/tc39/proposals/issues
+    about: TC39 manages the ECMAScript specification
```

Comment:
I don't think this is the right place. @ljharb

Reply:
If the issue is with the specification, the link should be https://github.com/tc39/ecma262/issues. But if it is a feature request, it should go ahead to https://es.discourse.group/.

Reply:
Does it really happen often that folks come to our issue tracker with problems that they _know_ are due to the language specification and not to Node.js?

Reply:
Peaking at https://github.com/nodejs/node/issues?q=is%3Aissue+label%3A%22wrong+repo%22+tc39, I think it might not be so frequent.

Reply:
I can remove it, but I was thinking that users come here when they have a suggestion for things Node.js doesn't control, like Arrays. I wasn't sure whether to put a link to TC39 or V8?

Reply:
I don't recall many folks having "issues with the specification" and reporting them to us. Some feature requests that should go to TC39/W3C/…, sure, but that's quite infrequent.